assets/sass/patterns/molecules/meta.scss

```diff
@@ -29,6 +29,12 @@
     content: "\00a0\2219\00a0";
     font-weight: bold;
   }
+
+  @media only all and (max-width: 480px) {
```

Comment:
Can we use the variable for the width please

Reply:
A variable is used now for the width.